roles/node/tasks/main.yml

```diff
@@ -1,4 +1,7 @@
 ---
+- name: Include setup_haproxy task
+  ansible.builtin.include_tasks: setup_haproxy.yml
```

Comment:
Please separate this as a different role, you can use default file locations, but we should separate it, it is an entirely different service, mixing it with node role isnt right